proto/validate/validate.proto

```
@@ -0,0 +1,862 @@
+  // Setting to false will enable a looser validations that only disallows
+  // \r\n\0 characters, which can be used to bypass header matching rules.
+  optional bool strict = 25 [default = true];
+
+  // IgnoreEmpty specifies that the validation rules of this field should be
+  // evaluated only if the field is not empty
+  optional bool ignore_empty = 26;
+}
+
+// WellKnownRegex contain some well-known patterns.
+enum KnownRegex {
+  UNKNOWN = 0;
```

Buf (1.47.2)

Enum value name "UNKNOWN" should be prefixed with "KNOWN_REGEX_".

(ENUM_VALUE_PREFIX)